aws-lc-rs/src/unstable.rs

```diff
@@ -9,4 +9,6 @@
 //! Features contained within this module, or child modules are subject to changes, relocation,
 //! or removal across minor releases, and thus are not subject to semantic versioning policies.
 
+#[cfg(not(feature = "fips"))]
+pub mod kdf;
```

Comment:
Would it be better to have the one-shot functions (`sskdf_hmac`, `sskdf_digest`, and `kbkdf_ctr_hmac`) fail at runtime, instead of having these functions disappear when the "fips" feature is enabled?

Granted this is the "unstable" module. The choice we have is between breaking at compile-time or breaking at runtime.  The former is more often preferable for consumers, but the latter seems the better match to a "features should be additive" policy.